src/app/visual/datepicker/datepicker-visual.component.html

```diff
@@ -83,7 +83,6 @@
         [minDate]="minDate"
         [skyDatepickerNoValidate]="noValidate"
         [startingDay]="startingDay"
-        [disabled]="disabled"
```

Comment:
Cleaned up this visual test so we would no longer get the console warning messages.